src/CVENix/Matching.hs

```diff
@@ -1,20 +1,35 @@
 module CVENix.Matching where
 
 import CVENix.SBOM
+import CVENix.Examples
 import Data.Maybe
 import Data.Text (Text)
+import qualified Data.Map as Map
 import qualified Data.Text as T
 
-match :: SBOM -> [Text] -> IO ()
+data Match = Match
+  { _match_pname :: Text
+  , _match_drv :: Text
+  , _match_advisory :: Advisory
+  }
+
+instance Show Match where
```

Comment:
Don't define an instance for show like this, if we want stuff "pretty printed" outputs we should use prettyprinter instead https://hackage.haskell.org/package/prettyprinter or just handwrite things that output the way we expect

add this after the last `}` for Match 
```haskell
deriving (Show, Eq, Ord)
```

Reply:
OK, moved the formatting to `match` for now :+1: